chains/evm/contracts/pools/Lombard/LombardTokenPool.sol

```diff
@@ -16,34 +18,80 @@ import {SafeERC20} from "@openzeppelin/contracts@4.8.3/token/ERC20/utils/SafeERC
 /// the pool performs validation, rate limiting, accounting and event emission.
 /// IPoolV2.lockOrBurn forwards tokens to the verifier.
 /// IPoolV2.releaseOrMint does not move tokens, _releaseOrMint is a no-op.
-/// TODO: Add explicit V1 support/backwards compatibility.
+/// IPoolV1.lockOrBurn and IPoolV1.releaseOrMint make this pool backwards compatible with old lanes.
 contract LombardTokenPool is TokenPool, ITypeAndVersion {
   using SafeERC20 for IERC20;
   using SafeERC20 for IERC20Metadata;
 
   error ZeroVerifierNotAllowed();
   error OutboundImplementationNotFoundForVerifier();
+  error ZeroBridge();
+  error ZeroLombardChainId();
+  error PathNotExist(uint64 remoteChainSelector);
+  error InvalidMessageVersion(uint8 expected, uint8 received);
+  error RemoteTokenMismatch(bytes32 bridge, bytes32 pool);
+  error InvalidReceiver(bytes receiver);
+  error ChainNotSupported(uint64 remoteChainSelector);
+  error InvalidAllowedCaller(bytes allowedCaller);
+  error ExecutionError();
+  error HashMismatch();
 
-  event LombardVerifierSet(address indexed verifier);
+  /// @param remoteChainSelector CCIP selector of destination chain.
+  /// @param lChainId The chain id of destination chain by Lombard Multi Chain Id conversion.
+  /// @param allowedCaller The address of TokenPool on destination chain allowed to handle GMP message.
+  event PathSet(uint64 indexed remoteChainSelector, bytes32 indexed lChainId, bytes32 allowedCaller);
```

Comment:
For PathSet & PathRemoved, we should specify if Lombard depends on them & how they are used given that they are named this way for a reason.